lib/Runtime/Library/TypedArray.cpp

```diff
@@ -46,6 +46,44 @@ namespace Js
     {
     }
 
+    Var TypedArrayBase::CreateNewInstanceFromObj(RecyclableObject *object, ScriptContext *scriptContext, uint32 elementSize, PFNCreateTypedArray pfnCreateTypedArray)
+    {
```

Comment:
Refactored the creation of list in the new rev.
The loop for populating newly created typed array behave differently from EntryForm() and therefore is left separate.